frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionEdit/ConstraintAccordionEdit.tsx

```diff
@@ -0,0 +1,278 @@
+import { useCallback, useEffect, useState } from 'react';
```

Comment:
This file is copied (And it's entire subtree), to keep a strict isolation between the new functionality and the old functionality.